pysat/tests/test_orbits.py

```diff
@@ -769,7 +699,10 @@ def teardown(self):
         del self.testInst, self.stime
 
 
-class TestOrbitsGappyData2(TestGeneralOrbitsMLT):
+class TestOrbitsGappyData2(TestOrbitsGappyData):
```

Comment:
I think the gappydata2 class came later after a bug was found. I don't think it should be running the TestOrbitsGappyData tests.

This class is checking if the first orbit is missing all data, and the first orbit sample starts on the day. Also has a single orbit gap unrelated to daybreak. I think we just need to do a standard iterate forward 20 orbits starting from 2009, 1, back 40, forward 20 type thing here.